src/graphics/device.js

```diff
@@ -902,6 +903,20 @@ Object.assign(pc, function () {
             this.grabPassTexture = grabPassTexture;
         },
 
+        updateGrabPass: function () {
+            var gl = this.gl;
+
+            var format = this.grabPassTexture._glFormat;
+            var source = (this.renderTarget && this.renderTarget.colorBuffer) || this.canvas;
+            gl.copyTexImage2D(gl.TEXTURE_2D, 0, format, 0, 0, source.width, source.height, 0);
+        },
+
+        unnitializeGrabPass: function () {
```

Comment:
Uninitialize doesn't have two 'n's, as far as I'm aware. However, shouldn't this instead be called destroyGrabPass to be more in line with other API naming conventions?

Reply:
I thought about naming it `destroyGrabPass` but then `initializeGrabPass` should probably also be renamed `createGrabPass` for symmetry. The only problem I see with that is it may give the impression to the reader that `createGrabPass` will return an object and `destroyGrabPass` takes a value and both are "pure" functions without side-effects.